fast_plotter/postproc/__main__.py

```diff
@@ -19,7 +19,7 @@ def make_parser():
                         help="A yaml to configure the post-processing step")
     parser.add_argument("-o", "--outdir", default=".",
                         help="The name of the output directory")
-    parser.add_argument("-V", "--value-cols", default=r"(.*sumw2?|n)",
+    parser.add_argument("-V", "--value-cols", default=r"(.*sumw2?|/^n$/)",
```

Comment:
Have you tested this? I think the second part of the group should fail due to the forward slashes. They have meaning in regexes in sed or vim, and a similar but slightly different meaning for e.g. javascript and java. As it is, I don't think the regex would match a column just called `n`.

The current regex:
```python
In [17]: re.findall(r"(.*sumw2?|/^n$/)", "test:sumw")
Out[17]: ['test:sumw']

In [18]: re.findall(r"(.*sumw2?|/^n$/)", "n")
Out[18]: []
```
Without the slashes:
```python
In [19]: re.findall(r"(.*sumw2?|^n$)", "n")
Out[19]: ['n']

In [20]: re.findall(r"(.*sumw2?|^n$)", "test:sumw")
Out[20]: ['test:sumw']
```
```suggestion
    parser.add_argument("-V", "--value-cols", default=r"(.*sumw2?|^n$)",
```